widgetastic_manageiq/__init__.py

```diff
@@ -4356,3 +4356,103 @@ def fill(self, value):
             return False
         self._all_item_elements[value or self.selected_item].click()
         return True
+
+
+class MigrationPlanRequestDetailsList(Widget):
+    """Represents the list of Migration Plan Request Details."""
+
+    ROOT = ParametrizedLocator('.//div[contains(@class,{@list_class|quote})]')
+    ITEM_LOCATOR = ParametrizedLocator('./div[contains(@class,"list-group-item")]')
```

Comment:
Please, correct xpath. it should be like ".//div[contains(@class,'list-group-item')]"

Reply:
It does not work with `.//`, hence used, `./`